### DISCUSSION TITLE
?is it possible to add a modell that turns pixelated/blurry photos to reference?

### DISCUSSION THREAD

**Discussion by yesbroc** [lllyasviel/ControlNet#419] [Ideas]
like upscaling with controlnnet magic, but this time the upscaler knows what you have in mind. say youre face is a bit blurred so you crop your face, put it through controlnet, tell it you have a beard and glasses and voila.

**Comment by geroldmeisinger** [comment]
this has been done in controlnet tile https://github.com/lllyasviel/ControlNet-v1-1-nightly
depending on your specific usecase you might also be interested in https://civitai.com/models/102482/pixelnet-controlnet-for-pixel-art but I have never tried this controlnet myself and don't know the quality
